app/src/main/java/com/example/gfood/screen/listmeal/ListMealFragment.kt

```diff
@@ -5,10 +5,40 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.GridLayoutManager
 import com.example.gfood.R
 import com.example.gfood.data.model.Category
+import com.example.gfood.data.model.Meal
+import com.example.gfood.data.source.remote.ListMealRemoteDataSource
+import com.example.gfood.data.source.repository.ListMealRepository
+import com.example.gfood.screen.listmeal.adapter.ListMealAdapter
+import com.example.gfood.screen.listmeals.ListMealConstract
+import com.example.gfood.screen.main.MainActivity
+import com.example.gfood.utils.loadImage
+import kotlinx.android.synthetic.main.fragment_list_meal.*
 
-class ListMealFragment : Fragment() {
+class ListMealFragment : Fragment(), ListMealConstract.View {
+
+    private val ARGUMENT_CATEGORY = "ARGUMENT_CATEGORY"
```

Comment:
xem lại convention
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#210-string-constants-naming-and-values